src/iterable/zip.js

```diff
@@ -0,0 +1,35 @@
+import {map} from './map'
+
+/**
+ * unpacks an Iterable of Iterables and yields an array of items
+ *
+ * @this {Iterable<any>}
+ * @ntime O(n)
+ * @dspace O(n)
+ * @example Basic Usage
+ *
+ * ```javascript
+ * [ [1,2,3] , [4,5], [6,7] ]::zip() yields  [1,4,6] , [2,4,7]
+ * ```
+*/
```

Comment:
the JSDoc block is attached to `generatorify` instead of `zip` which is why the build fails. :P The build requires all exports to have a leading JSDoc block.